### PR TITLE
Fix opencl check in configure.ac

### PR DESCRIPTION
**Pull request by talregev**
Investigating issue in Ubuntu and Debian in installation process #4252

### CODE REVIEW

**Comment by talregev** [comment]
My PR is pass the Ubuntu/Debian check:
https://code.launchpad.net/~costamagnagianfranco/+recipe/boinc-upstream-daily

File: m4/ax_check_cl.m4 [hunk 1]
```diff
@@ -1,120 +1,65 @@
...
-
-AS_IF([test "X$ax_cv_check_cl_libcl" = Xno],
-      [no_cl=yes; CL_CFLAGS=""; CL_LIBS=""],
-      [CL_LIBS="$ax_cv_check_cl_libcl $CL_LIBS"])
-AC_LANG_POP([C])
+# OpenCL support
+AC_DEFUN([AX_CHECK_CL],
+[AC_ARG_WITH([opencl],
+    [AS_HELP_STRING([--with-opencl],
+        [build OpenCL versions of SHOC benchmark programs @<:@default=check@:>@])],
```
Comment (ChristianBeer) [inline m4/ax_check_cl.m4:6]: Needs to be adjusted I think.

Comment (talregev) [reply m4/ax_check_cl.m4:6]: How? I am open to suggestion. 
It it the older check better?

Comment (talregev) [reply m4/ax_check_cl.m4]: @ChristianBeer I add a new indentation.

Comment (ChristianBeer) [reply m4/ax_check_cl.m4]: I meant for you to change the help string itself. This option toggles the opencl apps of BOINC and not the "SHOC benchmark program".

Comment (talregev) [reply m4/ax_check_cl.m4]: @ChristianBeer Thank you! you are right.
I change the help text.

**Comment by LocutusOfBorg** [comment]
yay thanks for fixing it!

**Review by AenBleidd — APPROVED** [review]

**Pull request merged**